web/marketing/css/style.css

```diff
@@ -38,4 +129,49 @@ body{
 
 .toggler:hover{
     cursor:pointer;
+}
+
+#foModalLabel, 
+#roModalLabel{
+    font-family: 'Poppins' !important;
```

Comment:
why `!important` if it's using id selector?

Reply:
I forgot to remove it after I got the static asset handler working. Just pushed the changes.